blueprints/schema.json

```diff
@@ -6412,6 +6412,7 @@
                             "authentik_core.add_groupsourceconnection",
                             "authentik_core.add_propertymapping",
                             "authentik_core.add_provider",
+                            "authentik_core.add_session",
```

Comment:
How do I avoid this?

Reply:
`default_permissions = []` in the Object's Meta class